TrainingTask/GameLogic.cs

```diff
@@ -0,0 +1,328 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace TrainingTask
+{
+    public class GameLogic
+    {
+        private MyCar myCar = new MyCar();
```

Comment:
All default values should be set in a constructor.

Reply:
fixed.